line-colors.csv

```diff
@@ -81,6 +81,34 @@ ooevv-oebb,S2,osterreichische-bundesbahnen,4-81-2-1589920-5372014,#0098a1,#fffff
 ooevv-oebb,S3,osterreichische-bundesbahnen,4-81-3-1589920-5372014,#342980,#ffffff,,rectangle-rounded-corner
 ooevv-oebb,S4,osterreichische-bundesbahnen,4-81-4-1589920-5372014,#a4ba00,#ffffff,,rectangle-rounded-corner
 ooevv-stern-hafferl,S5,stern-hafferl-verkehrs-gmbh,4-810008-5,#e2007a,#ffffff,,rectangle-rounded-corner
+rmv-db-sbahn,S1,db-regio-ag-s-bahn-rhein-main,4-800528-1,#009edd,#ffffff,,pill
+rmv-db-sbahn,S2,db-regio-ag-s-bahn-rhein-main,4-800528-2,#ff2e17,#ffffff,,pill
+rmv-db-sbahn,S3,db-regio-ag-s-bahn-rhein-main,4-800528-3,#00b098,#ffffff,,pill
+rmv-db-sbahn,S4,db-regio-ag-s-bahn-rhein-main,4-800528-4,#ffc734,#2c2e35,#2c2e35,pill
+rmv-db-sbahn,S5,db-regio-ag-s-bahn-rhein-main,4-800528-5,#95542a,#ffffff,,pill
+rmv-db-sbahn,S6,db-regio-ag-s-bahn-rhein-main,4-800528-6,#ff7322,#ffffff,,pill
+rmv-db-sbahn,S7,db-regio-ag-s-bahn-rhein-main,4-800528-7,#214d36,#ffffff,,pill
+rmv-db-sbahn,S8,db-regio-ag-s-bahn-rhein-main,4-800528-8,#88c946,#ffffff,,pill
+rmv-db-sbahn,S9,db-regio-ag-s-bahn-rhein-main,4-800528-9,#872996,#ffffff,,pill
```

Comment:
Kannst du die neuen EVU's und deine Quellen bitte noch in die `sources.json` eintragen, sodass die Validation durchgeht?
`unknown operator rmv-db-sbahn in row 84`

Reply:
na klar, ich hoffe das passt so (ist mein erster pull request ever, hoffe ich hab nichts kaputt gemacht 😬)

(Allerdings möchte ich darauf hinweisen, dass unter https://github.com/traewelling/line-colors#contributing explizit steht, dass man nix mit der sources.json machen soll, das müsste dann wohl geändert werden)